src/common/ActionMenu/ActionsMenu.tsx

```diff
@@ -24,27 +28,34 @@ const ActionsMenu = ({ actions }:Props) => {
           size="icon"
           className="focus:outline-none focus-visible:ring-0 active:outline-none active:ring-0"
         >
-
-          <DotsHorizontalIcon className="w-5 h-5" aria-label='icon' />
+          <DotsHorizontalIcon className="w-5 h-5" aria-label="icon" />
         </Button>
       </DropdownMenuTrigger>
       <DropdownMenuContent
-  align="end"
-  className="w-48 bg-white border rounded shadow-lg"
->
-  {actions.map((action, index) => (
-    <DropdownMenuItem
-      key={action.id}
-      onClick={action.onClick}
-      className="cursor-pointer px-4 py-2 hover:bg-gray-100 focus:outline-none focus-visible:ring-0 flex items-center space-x-2" 
-    >
-      {action.icon}
-      <span>{action.title}</span>
-    </DropdownMenuItem>
-  ))}
-</DropdownMenuContent>
+        align="end"
+        className="w-48 bg-white border rounded shadow-lg"
+      >
+        {actions.map((action, index) => (
+          <DropdownMenuItem
+            key={action.id}
+            onClick={action.onClick}
+            className="cursor-pointer px-4 py-2 hover:bg-gray-100 focus:outline-none focus-visible:ring-0 flex items-center space-x-2"
+          >
+            {action.icon}
+            <span
+              className={`${
+                action.title === "Delete" || action.title === "Reject"
```

Comment:
@nadirhussain786  we have enums for `reject` and `delete`

Reply:
Fixed